### DISCUSSION TITLE
General error : '"/mysqldump"' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by Abdullah-Iftikhar** [spatie/db-dumper#165] [General]
General error: '"/mysqldump"' is not recognized as an internal or external command, operable program or batch file.

I am using this for MYSQL and facing the above error.

Is there any solution to resolve this issue? I have tried a lot from google but not working for me. The file is created in the public directory but empty.

I want to download the database backup.

Same error. Not received any answer yet.
https://laravelquestions.com/2021/08/11/spatie-db-dumper-package-doesnt-recognize-mysqldump/

**Comment by tahirkeskin** [comment]
Hello,

You need to install the mysqldump package in the server.

For Example (Ubuntu);

`sudo apt-get install -y mysql-client`

Have a nice day.

**Reply by itsrexb** [reply]
In addition, you need to specify your mysqldump binary path. See https://spatie.be/docs/laravel-backup/v8/installation-and-setup#content-dumping-the-database
